src/syntax/parser.rs

```diff
@@ -376,7 +377,7 @@ impl<TokType: Tokenizer + Iterator<Item=Tokens>> Parser<TokType> {
 
         let expr = Expr::FnDecl(fn_name, args, return_type, definition);
 
-        return Some(ExprWrapper::default(expr));
+        Some(ExprWrapper::default(expr))
```

Comment:
I can put it back if you really want. I changed it because it didn't feel rusty. I personally see a function definition as its own `{}` block just like an if, so meh.